tests/sharingTest.js

```diff
@@ -168,6 +295,51 @@ fdescribe('Main: Currently testing file/folder sharing,', function () {
 
       // needs to be double checked
       describe('adding password,', function () {
+        beforeAll(async function (done) {
+          const additionalBodyElement = '  <share_with>***redacted***</share_with>\n' +
+                  '  <share_with_displayname>***redacted***</share_with_displayname>\n'
+
+          const promises = []
+          promises.push(setGeneralInteractions(provider))
+          promises.push(provider.addInteraction({
+            uponReceiving: 'a request to update a public link share',
+            withRequest: {
+              method: 'PUT',
+              path: Pact.Matchers.regex({
+                matcher: '.*\\/ocs\\/v1\\.php\\/apps\\/files_sharing\\/api\\/v1\\/shares\\/\\d+$',
+                generate: '/ocs/v1.php/apps/files_sharing/api/v1/shares/9'
+              }),
+              headers: validAuthHeaders,
+              body: 'password=test123'
+            },
+            willRespondWith: {
+              status: 200,
+              headers: {
+                'Access-Control-Allow-Origin': origin
+              },
+              body: '<?xml version="1.0"?>\n' +
+                '<ocs>\n' +
+                ocsMeta('ok', '100') +
+                ' <data>\n' +
+                shareResponseOcsData(3, 9, 1, '/testFolder') +
+                '  <share_with>***redacted***</share_with>\n' +
+                '  <share_with_displayname>***redacted***</share_with_displayname>\n' +
+                '  <name>..ffentlicher Link</name>\n' +
```

Comment:
why the dots?

Reply:
This is exactly what the name is when run in core server. `Öffentlicher` is shown as `..ffentlicher`

Reply:
also when you try to query it with curl?
My guess would be that wireshark did not display the `Ö` correctly. I can check tomorrow

Reply:
fixed